repos/system_upgrade/el7toel8/actors/vimcheck/actor.py

```diff
@@ -0,0 +1,82 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import InstalledRedHatSignedRPM, VimModel, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+vimconfigs = {
+              'vim-minimal': '/etc/virc',
+              'vim-enhanced' : '/etc/vimrc'
+              }
+
+changes = [
+            'let skip_defaults_vim=1',
+            'set t_BE='
+          ]
+
+
+
+class VimCheck(Actor):
+    """
+    Actor for checking if vim is installed and if vimrc file exists.
+    """
+
+    name = 'vim_check'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report, VimModel)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def checkconfigs(self, packages_dict):
+      """
```

Comment:
wrong intendation. This should eventually even crash in some cases. intendation is 4x space for every nested block. Fix intendation everywhere. It's broken on many places.